### PR TITLE
RELEASE PR 1.1.0 "Picometre"

### PR DESCRIPTION
**Pull request by apeltzer**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/nanostring/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/nanostring _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow_schema.json [hunk 1]
```diff
@@ -42,6 +42,42 @@
...
+            "title": "Processing options",
+            "type": "object",
+            "description": "Define how the pipeline should process the data.",
+            "default": "",
+            "fa_icon": "fas fa-exchange-alt",
+            "properties": {
+                "heatmap_genes_to_filter": {
+                    "type": "string",
+                    "description": "Path to yml file (list, one item per line) to specify which genes should be used for the gene-count heatmap.",
+                    "default": "None"
```
Comment (maxulysse) [inline nextflow_schema.json:55]:
```suggestion
                    "description": "Path to yml file (list, one item per line) to specify which genes should be used for the gene-count heatmap."
```

File: nextflow_schema.json [hunk 2]
```diff
@@ -42,6 +42,42 @@
...
+            "properties": {
+                "gene_score_method": {
+                    "type": "string",
+                    "default": "PLAGE",
+                    "fa_icon": "fas fa-adjust",
+                    "description": "This selects the algorithm for computing the respective gene score. Default is 'PLAGE'."
+                },
+                "gene_score_yaml": {
+                    "type": "string",
+                    "default": "None",
```
Comment (maxulysse) [inline nextflow_schema.json:74]:
```suggestion
```

File: bin/compute_gene_heatmap.R [hunk 3]
```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env Rscript
```
Comment (maxulysse) [inline bin/compute_gene_heatmap.R:1]: no license / author for the script?

**Review by maxulysse — CHANGES_REQUESTED** [review]
just remove the default to none in the schema and we're good to go

File: bin/compute_gene_heatmap.R (hunk 3 above)
Comment (apeltzer) [reply bin/compute_gene_heatmap.R:1]: Its us, we will submit the paper immediately after this is released and after all its just part of the pipeline :)

**Review by maxulysse — APPROVED** [review]
LGTM

File: CITATIONS.md [hunk 4]
```diff
@@ -14,6 +14,14 @@
   > Ewels P, Magnusson M, Lundin S, Käller M. MultiQC: summarize analysis results for multiple tools and samples in a single report. Bioinformatics. 2016 Oct 1;32(19):3047-8. doi: 10.1093/bioinformatics/btw354. Epub 2016 Jun 16. PubMed PMID: 27312411; PubMed Central PMCID: PMC5039924.
 - [NACHO](https://pubmed.ncbi.nlm.nih.gov/31504159/)
   > Canouil M, Bouland GA, Bonnefond A, Froguel P, Hart L, Slieker R. NACHO: an R package for quality control of NanoString nCounter data. Bioinformatics. 2019 Aug 23;36(3), 970-971. doi: 10.1093/bioinformatics/btz647. PubMed PMID: 31504159; PubMed Central PMCID: PMC9883715.
+- [GSVA](https://link.springer.com/article/10.1186/1471-2105-14-7) - Hänzelmann, Sonja, Robert Castelo, and Justin Guinney. "GSVA: gene set variation analysis for microarray and RNA-seq data." BMC bioinformatics 14 (2013): 1-15.
```
Comment (christopher-mohr) [inline CITATIONS.md:17]:
```suggestion
- [GSVA](https://link.springer.com/article/10.1186/1471-2105-14-7)
   > Hänzelmann, Sonja, Robert Castelo, and Justin Guinney. "GSVA: gene set variation analysis for microarray and RNA-seq data." BMC bioinformatics 14 (2013): 1-15.
```

File: nextflow.config [hunk 5]
```diff
@@ -212,7 +220,7 @@ manifest {
     description     = """A Nanostring nCounter analysis pipeline"""
     mainScript      = 'main.nf'
     nextflowVersion = '!>=23.04.0'
-    version         = '1.0.0'
+    version = '1.1.0dev'
```
Comment (christopher-mohr) [inline nextflow.config:223]:
```suggestion
    version = '1.1.0'
```

**Pull request merged**